lib/vecmat.gi

```diff
@@ -1411,12 +1411,12 @@ local sf, rep, ind, ind2, row, i,big,l;
   else
     if not IsField(field) then
       # not a field
-      return Immutable(matrix);
+      return matrix;
```

Comment:
I have not touched this line. Seems that this very recent change got lost when I ported my fix to the latest  master branch. I will add back that `Immutable`.

I don't understand what this case means. What are valid arguments for this function?